src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Performance/RecommendCaseInsensitiveStringComparison.Analyzer.cs

```diff
@@ -199,7 +199,14 @@ private void AnalyzeCompilationStart(CompilationStartAnalysisContext context)
                     return;
                 }
 
-                if (caseChangingInvocation.Parent is not IInvocationOperation diagnosableInvocation)
+                // Ignore parenthesized operations
+                IOperation? ancestor = caseChangingInvocation.Parent;
+                while (ancestor is not null and IParenthesizedOperation)
+                {
+                    ancestor = ancestor.Parent;
+                }
```

Comment:
There is already an `WalkUpParentheses` extension method.

Reply:
Good to know. Let me use it.